index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://code.jquery.com/jquery-3.6.4.min.js" defer></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <section class="homepage">
+        <nav id="navbar">
+            <div><a href="#" class="mylogo">My logo</a></div>
+            <div class="hamburger"><img src="Assets/files/hamburger.png" alt="loading"></div>
+        </nav>
+        <div class="home-body">
```

Comment:
- [ ] I will recommend that you use the `<section>` tag to group your headline section because the section tag is used for the thematic grouping of elements that are standalone, so any combination of elements that has a heading and makes sense on its own if extracted from the page should be wrapped in a <section> tag 👍🏽